CHANGELOG.md

```diff
@@ -1,4 +1,57 @@
 # nf-core/atacseq: Changelog
 
-## v1.0dev - [date]
-Initial release of nf-core/atacseq, created with the [nf-core](http://nf-co.re/) template.
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [1.1.0] - 2019-11-01
```

Comment:
```suggestion
## [1.1.0] - 2019-11-05
```

Maybe 5th?

Reply:
Ah, that's been changed in the open PR to `dev` :+1:

Reply:
I'll merge that last minute before the merge to `master`